spec/jobs/s3_east_delivery_job_spec.rb

```diff
@@ -0,0 +1,9 @@
+describe S3EastDeliveryJob, type: :job do
```

Comment:
if you run `be rspec spec/job/s3_east_delivery_job_spec.rb` by it self, it fails. is it bc this is inheriting from s3_west, so it needs to run s3_west_spec before the s3_east_spec is ran?

Reply:
ah, just forgot to require `rails_helper`, thx